### DISCUSSION TITLE
[Work-around solved issue!] One of my projects displays no UI at all on clicking the Playtime toolbar button (but Reaper menubar goes white to indicate another window has opened)

### DISCUSSION THREAD

**Discussion by FakenameMusic** [helgoboss/helgobox#1208] [General]
All other/new projects are working fine.
One project  (which I have unfortunately been working with for a while)  does not show the Playtime UI at all when I click the Playtime button. However, above the menu bar in Reaper turns from blue to white indicating that another window has launched on top.
I can't interact with this invisible window at all, although I can still launch / stop clips via my Launchpad. The whole Playtime clip matrix is working just as it should, controls mapped with Relearn are also fine. I just cant bring up the Playtime UI window to do anything more than I have already done, so I am stuck and cannot continue with this song.

There is nothing behind the Reaper window, I've checked.

I don't know if this is related, but I was originally working with this project with the Playtime window sent to a 2nd Windows monitor hosted on my iPad via Spacedesk. But still, regardless of the 2nd "monitor" being connected or not, I cannot get the UI to appear.

**Comment by helgoboss** [comment]
It could indeed be related to Helgobox trying to open the app window at a certain position that's off-screen. Can you please file an issue?

As a workaround:

- Export to clipboard => Export instance as JSON
- Copy the result into a text editor
- Look for something like appSettings and window position (can't check now the exact text). Change the position accordingly.
- copy and import back into ReaLearn

**Comment by FakenameMusic** [comment]
Thanks for your reply Benjamin, I have filed an issue.

[EDIT] I have found "Export to clipboard".
I will do as you advised for the work around.
Thanks!

**Comment by FakenameMusic** [comment]
That workaround has worked, thanks Benjamin.

For anyone else with this issue who needs a work around:

- Click the FX button on your Playtime Channel

- Select the "Helgobox: Relearn and Playtime" effect in the left pane.

- Click on "Export to Clipboard" the "Export Instance To JSON" when it pops up.
![image](https://github.com/user-attachments/assets/c973f749-22f1-4d4a-93f2-2746e908f83b)

- Open a text editor (like Notepad in WIndows or TextEdit in Mac OS) and Paste in the copied text. (You will not have to save this document - you will work entirely with the "copy/paste" clipboard)

- Look for this part of the text:

 

> "customData": {
>       "app": {
>         "fullScreen": false,
>         "windowBounds": {
>           "height": 554.4,
>           "width": 828.8,
>           "x": 50,
>           "y": 50

The x/y coordinate at the end represents the position of the Playtime UI window. In my case it was well outside the resolution of  my monitor so I changed the digits to 50, 50 (as seen above) to bring it to the top left of my screen. Make sure not to delete the comma (,) after the first digit.

- When you change the numbers select and copy *all* of the text in the document.

- Return to the Playtime FX window and click "Import from Clipboard"

![image](https://github.com/user-attachments/assets/e1814295-53e7-4851-8381-188279e3a338)

- You will now get a pop up "warning" asking are you sure. If you are sure, go ahead and click "Yes".

![image](https://github.com/user-attachments/assets/26bde95e-cdfb-43c7-8356-5fda4c2d7c9a)

You don't need to look to "save" or "apply" anything here - you can just close the window. As long as you Save the Reaper project it will be fine the next time that you open it.

- Now go back to the Reaper UI and click the Playtime button in the toolbar.

- You should now see the Playtime UI pop up. If you do not, try making the numbers that you changed smaller again. 0,0 will put it almost at the very top left of your screen if you are using 1 monitor.

- Now feel free to, once again, make awesome fun things and live a very pleasurable life. 🌴😎🌴

**Comment by helgoboss** [comment]
Will be fixed in 2.16.9
